input/events/piano-night-social.md

```diff
@@ -3,7 +3,7 @@ title: Piano Night Social
 times:
   - time: 2021-07-28 19:30
 image: /assets/SedosWebsite-Banner-sechoir.jpg
-venue: bridewell
+venue: Bridewell Theatre
```

Comment:
I _thought_ we were referencing the venue collection, so "key" should have just worked automatically, is that not the case?

Reply:
We previously did this: ```x["key"].ToString().Equals(venueName)``` - which didn't match with all of the newly created pages (I assume because of how was entered in the CMS looking at the pages) - but did match a load of previous shows. I _think_ this matches more closely with what we'll get from people who deal with the content - but happy to go back to something else.

Reply:
Okay, that's fine. I _tried_ to make it something key rather than user friendly based, but if this works I'm not too worried.

Reply:
I _think_ doing this will make us automatically get everything for the future. It would be _nice_ if we could generate links between things in the backend, but I'm not sure if the CMS is powerful enough to let us do those kind of relationships in the backend (I'll be honest - I haven't looked at the docs - but if there is, would consider doing that as part of a separate piece of work)